packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -18,6 +18,8 @@ export interface WizardStep {
   name: React.ReactNode;
   /** The component to render in the main body */
   component?: any;
+  /** The content to render in the drawer panel */
+  drawerPanelContent?: any;
```

Comment:
Nit: what do you think of tweaking the prop description to mention the `hasDrawer` prop, e.g. "The content to render in the drawer panel when hasDrawer is also passed in"? My thinking is just to connect the two props together more, so that consumers will know immediately they also need to pass `hasDrawer` in from the description.

Reply:
Good idea - definitely makes sense!